core/templates/dev/head/css/oppia.css

```diff
@@ -1491,6 +1491,7 @@ pre.oppia-pre-wrapped-text {
 .oppia-editor-card-section {
   padding: 20px 50px 20px 35px;
   word-wrap: break-word;
+  height: 200px;
```

Comment:
This is used throughout the codebase make sure this doesn't affect other parts of the UI components.

Reply:
This also makes the height of exploration card 200px. Should I go for inline style as in the previous commit.

Reply:
maybe handle this in the specific file with child selector? will that work?

Reply:
@DubeySandeep I feel that the inline style would be better option here as I have tried other possible methods but I am not getting the specific changes

Reply:
@DubeySandeep

Reply:
@Dharmesh-Poddar I would suggest you to try not using the inline style but if you think it won't be possible without that then please go ahead and do it!